system/admin.py

```diff
@@ -1,17 +1,10 @@
 from django.contrib import admin
-from .models import User, Equipment, Search, Reserved, Request, Vote, Log
+from .models import *
```

Comment:
__[flake8]__
* [F403] 'from models import *' used; unable to detect undefined names